internal/server/search.go

```diff
@@ -369,6 +372,16 @@ func (server *Server) cmdSearchArgs(
 				return
 			}
 			s.obj = geojson.Clip(s.obj, clip_rect, &server.geomIndexOpts)
+		case "object":
+			var obj string
+			if vs, obj, ok = tokenval(vs); !ok || obj == "" {
+				err = errInvalidNumberOfArguments
+				return
+			}
+			s.clipby, err = geojson.Parse(obj, &server.geomParseOpts)
+			if err != nil {
+				return
+			}
```

Comment:
This creates a hidden trap: t38 cannot do polygon/polygon intersection, so allowing `object` for clipby will fail when both sides of the comparison are polys